app/components/object-inspector/sort-properties.js

```diff
@@ -19,14 +19,20 @@ export default Component.extend({
    */
   sortedProperties: computed('sorted.length', function () {
     // limit arrays
-    if (this.get('isArray') && this.get('sorted.length') > 100) {
+    let props = A(this.get('sorted'));
+    if (this.isArray()) {
+      const item = props.findBy('name', 'length');
```

Comment:
What is this doing?

Reply:
This is to put the length property at the top, since otherwise it will be cutoff. Or it will look wierd at the end